release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -593,106 +593,94 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 In the table, features are marked with the following statuses:
 
-* *GA*: _General Availability_
-* *DEP*: _Deprecated_
-* *REM*: _Removed_
-
-//TODO: remove anything that has been REM since 4.9?
+* _General Availability_
+* _Deprecated_
+* _Removed_
 
 .Deprecated and removed features tracker
 [cols="3,1,1,1",options="header"]
 |====
-|Feature |OCP 4.9 |OCP 4.10 | OCP 4.11
+|Feature |4.10 |4.11 | 4.12
 
 |SQLite database format for Operator catalogs
-|DEP
-|DEP
-|DEP
+|Deprecated
+|Deprecated
+|Deprecated
 
 |`ImageChangesInProgress` condition for Cluster Samples Operator
-|DEP
-|DEP
-|DEP
+|Deprecated
+|Deprecated
+|Deprecated
 
 |`MigrationInProgress` condition for Cluster Samples Operator
-|DEP
-|DEP
-|DEP
-
-|Cluster Loader
-|DEP
-|REM
-|REM
-
-|Bring your own {op-system-base} 7 compute machines
-|DEP
-|REM
-|REM
-
-|Jenkins Operator
-|DEP
-|REM
-|REM
+|Deprecated
+|Deprecated
+|Deprecated
 
 |Grafana component in monitoring stack
-|-
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |Access to Prometheus and Grafana UIs in monitoring stack
-|
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |vSphere 6.7 Update 2 or earlier
-|DEP
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |vSphere 7.0 Update 1 or earlier
-|-
-|-
-|DEP
+|General Availability
+|Deprecated
+|Deprecated
 
 |Virtual hardware version 13
-|DEP
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |VMware ESXi 6.7 Update 2 or earlier
-|DEP
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |VMware ESXi 7.0 Update 1 or earlier
-|-
-|-
-|DEP
+|General Availability
+|Deprecated
+|Deprecated
 
 |Snapshot.storage.k8s.io/v1beta1 API endpoint
-|DEP
-|DEP
-|REM
-
-|Minting credentials for Microsoft Azure clusters
-|GA
-|REM
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |Persistent storage using FlexVolume
-|-
-|DEP
-|DEP
+|Deprecated
+|Deprecated
+|Deprecated
 
 |Automatic generation of service account token secrets
-|GA
-|GA
-|REM
+|General Availability
+|Removed
+|Removed
 
 |Removal of Jenkins images from install payload
-|GA
-|GA
-|REM
+|General Availability
+|Removed
+|Removed
+
+|CoreDNS wildcard queries for the `cluster.local` domain
+|General Availability
+|General Availability
+|Deprecated
+
+|Special Resource Operator (SRO)
+|General Availability
+|General Availability
+|Deprecated
 
```

Comment:
@opayne1 We need to add deprecation for the "old" hardware.  Since the list is quite long might be best to it up per architecture:  
```
|IBM POWER8 all models, IBM POWER9 (AC922, IC922, LC922) (ppc64le)
|General Availability
|General Availability
|Deprecated

|IBM z13 all models, LinuxONE Emperor, LinuxONE Rockhopper (s390x)
|General Availability
|General Availability
|Deprecated

|AMD64 (x86_64) v1 CPU
|General Availability
|General Availability
|Deprecated

```
Or maybe create a new multi-architecture table like you did for TP. 
```

.Multi-Architecture deprecated and removed features tracker
[cols="3,1,1,1",options="header"]
|====
|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12

|IBM POWER8 all models (ppc64le)
|General Availability
|General Availability
|Deprecated

|IBM IBM POWER9 AC922 (ppc64le)
|General Availability
|General Availability
|Deprecated

|IBM IBM POWER9 IC922 (ppc64le)
|General Availability
|General Availability
|Deprecated

|IBM IBM POWER9 LC922 (ppc64le)
|General Availability
|General Availability
|Deprecated

|IBM z13 all models (s390x)
|General Availability
|General Availability
|Deprecated

|IBM LinuxONE Emperor (s390x)
|General Availability
|General Availability
|Deprecated

|IBM LinuxONE Rockhopper (s390x)
|General Availability
|General Availability
|Deprecated

|AMD64 (x86_64) v1 CPU
|General Availability
|General Availability
|Deprecated
```

Reply:
Thank you for the suggestion @SNiemann15 I like the idea of breaking this table up too!